controllers/comment.controller.js

```diff
@@ -79,10 +84,32 @@ const addComment = async (req, res) => {
                 error: `There are no ${cityName} in list. For full list see: /api/cities/`,
             });
         }
-        await Comment.create({
+        const ifCommentInDb = await Comment.create({
```

Comment:
if variable ex: 'ifCommentInDb' shoud be bool
if variable is object, array - it is commentInDb and them you check is variable as you wish